### PR TITLE
Feature/1204231322750283 create docker image

### PR DESCRIPTION
**Pull request by sayyed-kamran**
The purpose of this PR is to give command to users to run peaq parachain in docker container that will connect to relaychain running in peaq development environment.

### CODE REVIEW

File: Dockerfile.old
Comment (sfffaaa) [inline Dockerfile.old:1]: What's the purpose of this file?

File: README.md [hunk 1]
```diff
@@ -181,24 +181,66 @@ A FRAME pallet is compromised of a number of blockchain primitives:
 First, install [Docker](https://docs.docker.com/get-docker/) and
 [Docker Compose](https://docs.docker.com/compose/install/).
 
-Then run the following command to start a single node development chain.
+Please use following command to run peaq-network-node parachian in the docker container connected with polkadot relaychain running in the PEAQ development environment.
```
Comment (sfffaaa) [inline README.md:184]: Do we have the document about how to create the docker image? Maybe in README.md or Slab?

File: Dockerfile.old
Comment (sayyed-kamran) [reply Dockerfile.old:1]: In the start,  there was already a file with the name Dockerfile, so I created a docker file with the name Dockerfile.parachain. But later on, @niazhussain  told me that the workflow defined to build docker image require the name of the file to be exactly "Dockerfile" so I renamed the file Dockerfile to Dockerfile.Old and rename my file from Dockerfile.parachain to DockerFile.

File: README.md (hunk 1 above)
Comment (sayyed-kamran) [reply README.md:184]: We do not have any such instruction in README.md. I checked on slap but I was not able to figure it out on slab too. So, in my view we do not have any document that explains how to create the docker image for peaq-network-node parachain.

File: Dockerfile.old
Comment (sfffaaa) [reply Dockerfile.old:1]: Okay, then do we need it or can we delete it? 
If we need that, should we change the name (Dockerfile.old) to a more meaningful name?

File: README.md (hunk 1 above)
Comment (sfffaaa) [reply README.md:184]: Got it. Then please create one document about that.

File: Dockerfile.old
Comment (sayyed-kamran) [reply Dockerfile.old:1]: I think we do not need "Dockerfile.old" any more. As the parachain_launch/peaq-node-builder also uses the docker files available in the scripts folder.

File: README.md (hunk 1 above)
Comment (sayyed-kamran) [reply README.md:184]: Oky Sure.

File: Dockerfile.old
Comment (sfffaaa) [reply Dockerfile.old:1]: I think so, but please double-check with Niaz.

Comment (sayyed-kamran) [reply Dockerfile.old:1]: I discussed with Niaz too, he also said to delete the file "Dockerfile.Old"

Comment (sayyed-kamran) [reply Dockerfile.old:1]: I discussed with @niazhussain and he is of the view that we should delete that file. So, I delete the Dockerfile.old.

**Review by sfffaaa — APPROVED** [review]

**Pull request merged**